### PR TITLE
fix(cli): escape char sequences for javascript comments

### PR DESCRIPTION
**Pull request by raymondfeng**
1. Escape `/*` and `*/` inside `description` for generated code comments.
2. Use `_requestBody` to avoid tslint shadowed name complaints

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [x] `npm test` passes on your machine
- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [x] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: packages/cli/generators/openapi/utils.js [hunk 1]
```diff
@@ -148,6 +153,9 @@ function escapeIdentifier(name) {
   if (!name.match(SAFE_IDENTIFER)) {
     name = _.camelCase(name);
   }
+  if (DECORATOR_NAMES.includes(name)) {
```
Comment (b-admike) [inline packages/cli/generators/openapi/utils.js]: Are these changes related to the escape comment fix? It might be good to separate them into a different commit if not.

File: packages/cli/test/unit/openapi/controller-spec.unit.js [hunk 2]
```diff
@@ -44,13 +44,13 @@ describe('openapi to controllers/models', () => {
...
-              '@param requestBody Customer to add',
-              '@param accessToken Access token',
+              '@param _requestBody Customer to add',
+              '@param accessToken Access token (/* access_token */)',
               '@returns customer response',
             ],
             decoration: "@operation('post', '/customers')",
             signature:
-              'async createCustomer(@requestBody() requestBody: Customer, ' +
+              'async createCustomer(@requestBody() _requestBody: Customer, ' +
```
Comment (jannyHou) [inline packages/cli/test/unit/openapi/controller-spec.unit.js]: wouldn't the name here be `customer` instead of `requestBody` or `_requestBody`

File: packages/cli/generators/openapi/utils.js (hunk 1 above)
Comment (raymondfeng) [reply packages/cli/generators/openapi/utils.js]: Refactored into two commits.

File: packages/cli/test/unit/openapi/controller-spec.unit.js (hunk 2 above)
Comment (raymondfeng) [reply packages/cli/test/unit/openapi/controller-spec.unit.js]: We use parameter names from the openapi spec instead of inferring it from the type.

Comment (jannyHou) [reply packages/cli/test/unit/openapi/controller-spec.unit.js]: I see.
Then it will always be `requestBody`, only parameters have a name in the OpenAPI spec.

File: packages/cli/test/unit/openapi/openapi-utils.unit.js [hunk 3]
```diff
@@ -41,4 +41,11 @@ describe('openapi utils', () => {
     expect(utils.escapePropertyName('customer_id')).to.eql('customer_id');
     expect(utils.escapePropertyName('customerid')).to.eql('customerid');
   });
+
+  it('escapes chars for comments', () => {
+    expect(utils.escapeComment('/* abc */')).to.eql('\\/* abc *\\/');
```
Comment (b-admike) [inline packages/cli/test/unit/openapi/openapi-utils.unit.js]: I wonder if we should escape them only if it's a complete comment like `/* .... */` instead of `/* ....` or `... */`. I'm thinking perhaps if it's a malformed/unfinished comment we can fail fast.

Comment (raymondfeng) [reply packages/cli/test/unit/openapi/openapi-utils.unit.js]: We should accept what is in the `description` as-is. It's not our responsibility to check the content of OpenAPI spec `description`s.

Comment (b-admike) [reply packages/cli/test/unit/openapi/openapi-utils.unit.js]: Okay that makes sense.
